### PR TITLE
Added isWithinNext and wasWIthinLast methods

### PR DESCRIPTION
**Pull request by lorenzo**

### CODE REVIEW

File: tests/DateTime/IsTest.php [hunk 1]
```diff
@@ -444,4 +444,50 @@ public function testIsThisYear($class)
...
+        $this->assertTrue((new $class('-1 second'))->wasWithinLast('1 second'));
+        $this->assertTrue((new $class('-1 day'))->wasWithinLast('1 week'));
+        $this->assertTrue((new $class('-1 week'))->wasWithinLast('2 week'));
+        $this->assertTrue((new $class('-1 second'))->wasWithinLast('10 minutes'));
+        $this->assertTrue((new $class('-1 month'))->wasWithinLast('13 month'));
+        $this->assertTrue((new $class('-1 seconds'))->wasWithinLast('1 hour'));
+        $this->assertFalse((new $class('-1 year'))->wasWithinLast('1 second'));
+        $this->assertFalse((new $class('-1 year'))->wasWithinLast('0 year'));
+        $this->assertFalse((new $class('-1 weeks'))->wasWithinLast('1 day'));
+        $this->assertTrue((new $class('-3 days'))->wasWithinLast('5'));
```
Comment (markstory) [inline tests/DateTime/IsTest.php]: This input value is pretty ambiguous. Should we even support this?

File: src/DateTimeTrait.php [hunk 2]
```diff
@@ -1833,6 +1833,46 @@ public function isBirthday(ChronosInterface $dt)
...
+     *
+     * @param string|int $timeInterval the numeric value with space then time type.
+     *    Example of valid types: 6 hours, 2 days, 1 minute.
+     * @return bool
+     */
+    public function isWithinNext($timeInterval)
+    {
+        $tmp = str_replace(' ', '', $timeInterval);
+        if (is_numeric($tmp)) {
+            $timeInterval = $tmp . ' days';
```
Comment (markstory) [inline src/DateTimeTrait.php]: I don't see any tests for this branch.

File: tests/DateTime/IsTest.php (hunk 1 above)
Comment (lorenzo) [inline tests/DateTime/IsTest.php]: I can remove

File: src/DateTimeTrait.php (hunk 2 above)
Comment (lorenzo) [inline src/DateTimeTrait.php]: That is actually the ambiguous value you wanted to remove!

**Comment by lorenzo** [comment]
@markstory Fixed your comments

**Pull request merged**